src/service/http-server.ts

```diff
@@ -234,7 +235,7 @@ export class HttpServer {
     const result = await this.browser.renderCSV(options);
 
     if (result.fileName) {
-      res.setHeader('Content-Disposition', `attachment; filename="${result.fileName}"`);
+      res.setHeader('Content-Disposition', contentDisposition(result.fileName));
```

Comment:
What happens if the file name is expected to contains special characters - will they be simply removed?